src/modules/asset/utils.ts

```diff
@@ -52,8 +52,7 @@ export async function getContentsCID(asset: RawAsset): Promise<Record<string, st
 
   for (let filePath of Object.keys(contents)) {
     const file = await makeContentFile(filePath, contents[filePath])
-    const fileCID: string = await getCID([{ path: path.basename(file.path), content: file.content, size: file.size }], false)
```

Comment:
Should we call the function getContentsHash?

Reply:
Good catch!